sdk/messaging/azservicebus/go.mod

```diff
@@ -3,21 +3,20 @@ module github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus
 go 1.16
 
 require (
-	github.com/Azure/azure-amqp-common-go/v3 v3.1.0
+	github.com/Azure/azure-amqp-common-go/v3 v3.2.0
```

Comment:
Also, is there an issue tracking removal of this dependency?  I know we talked about moving this to an internal package to be shared with EH.

Reply:
Created: https://github.com/Azure/azure-sdk-for-go/issues/15646